graph/db/graph.go

```diff
@@ -605,9 +573,6 @@ func (c *ChannelGraph) MarkEdgeZombie(chanID uint64,
 func (c *ChannelGraph) UpdateEdgePolicy(edge *models.ChannelEdgePolicy,
 	op ...batch.SchedulerOption) error {
 
-	c.cacheMu.Lock()
```

Comment:
don't we need the lock tho to access below to avoid race conditions tho ?

`if c.graphCache != nil {` ...

Reply:
so does this comment still hold  seems like we will not use it as described ?

```
// cacheMu guards any writes to the graphCache. It should be held
	// across the DB write call and the graphCache update to make the
	// two updates as atomic as possible.
	cacheMu sync.Mutex
```

Reply:
no that is why it is being removed here. We are "undoing" this behaviour and reverting to how it was before this was introduced.